sec_certs_page/templates/index.html.jinja2

```diff
@@ -152,20 +100,24 @@
                         <div class="feature-icon bg-primary bg-gradient">
                             <i class="fas fa-chart-line fa-fw"></i>
                         </div>
-                        <h3>Trend visualizations</h3>
-                        <p>Explore our analysis of the security certifications landscape as it evolves over time.
-                            The analysis results presented here are preliminary, for more, see our published
-                            <a href="{{ url_for("about", _anchor="research") }}">research</a>.</p>
+                        <h3>Trends visualization</h3>
+                        <p>
+                            Explore our analysis of the evolving security certifications landscape.
+                        </p>
+                        <p>
+                            We generate graphs from certification PDFs by collecting and processing them into machine-readable datasets, which we then publish.
+                            You can view example visualizations in the Analysis section. 
+                            The results presented here are preliminary - refer to our published 
```

Comment:
In this paragraph, graphs, visualizations and results are used interchangeably and I am confused about what that means. Also, graphs point more towards reference graphs, I would use plots to talk about our plots.